File: Practice Arrays/codigo_cs/19Candies.cs
using System;
using System.Linq;
using System.Collections.Generic;
public class Test19
{
    public static void Candies()
    {
        // your code goes here
        int t = int.Parse(Console.ReadLine());
        for (int _ = 0; _ < t; _++)
        {
            int n = int.Parse(Console.ReadLine());
            int[] arr = Console.ReadLine().Split().Select(int.Parse).ToArray();
            Dictionary<int, int> dict = new Dictionary<int, int>();
            bool isPossible = true;
            for (int i = 0; i < 2 * n; i++)
            {
                if (dict.ContainsKey(arr[i]))
                {
                    dict[arr[i]]++;
                }
                else
                {
                    dict.Add(arr[i], 1);
                }

                if (dict[arr[i]] > 2)
                {
                    isPossible = false;
                    break;
                }
            }

            if (isPossible)
            {
                Console.WriteLine("Yes");
            }
            else
            {
                Console.WriteLine("No");
            }

        }

    }
}

File: Practice Arrays/19Candies.py
t=int(input())
for _ in range(t):
    n=int(input())
    arr=list(map(int,input().split()))
    count={}
    isPossible=True
    for num in arr:
        if num in count:
            count[num]+=1
        else:
            count[num]=1
        if count[num]>2:
            isPossible=False
            break

    print("Yes" if isPossible else "No")